spring/src/main/java/io/seata/spring/annotation/GlobalTransactionalInterceptor.java

```diff
@@ -74,6 +74,33 @@
     private static ScheduledThreadPoolExecutor executor =
         new ScheduledThreadPoolExecutor(1, new NamedThreadFactory("degradeCheckWorker", 1, true));
 
+    //region GLOBAL_TRANSACTION_TIMEOUT
+
+    private static int GLOBAL_TRANSACTION_TIMEOUT;
+    private static int MIN_GLOBAL_TRANSACTION_TIMEOUT = DEFAULT_GLOBAL_TRANSACTION_TIMEOUT / 2;
+
+    static {
+        try {
```

Comment:
why not move to the constructor block?

Reply:
The GlobalTransactionalInterceptor is a single instance, so it is not necessary to put it in the constructor for the moment.